### PR TITLE
Machine updates

### PR DESCRIPTION
**Pull request by jannisborn**
2024-10-19 22:30:02 INFO ======Location differ joblog from 2024-10-19 22:30:02=======
2024-10-19 22:32:09 INFO A Christmas Story House is available again
2024-10-19 22:33:25 INFO Location Ohio (2/125): Changes in 1/138 machines found.
2024-10-20 00:07:28 INFO Busan Tower is available again
2024-10-20 00:07:36 INFO Location South Korea (111/125): Changes in 1/11 machines found.
2024-10-20 00:12:47 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-10-20 00:28:24 ERROR Found 3 problems that require manual intervention
2024-10-20 00:28:24 INFO ======Location differ completed at 2024-10-20 00:28:24=======
2024-10-20 00:28:25 INFO Detected change in server_locations.json - push to github

### CODE REVIEW

**Comment by jannisborn** [comment]
No website updates today!

**Comment by NinaWie** [comment]
New machine 7597 named Souvenir shop submitted.

**Comment by jannisborn** [comment]
2024-10-21 23:59:05 INFO ======Location differ joblog from 2024-10-21 23:59:05=======
2024-10-22 00:13:00 INFO 167/222: Found machine to be added: Gateway Clipper
2024-10-22 00:13:32 INFO Location Pennsylvania (11/125): Changes in 1/222 machines found.
2024-10-22 00:38:05 INFO 3/310: Found machine to be added: Dream Maker Station Route 66 Souvenir & Gift Shop From (Midpoint Cafe)
2024-10-22 00:38:05 INFO Midpoint Cafe and Gift Shop is currently unavailable
2024-10-22 00:38:31 INFO 48/310: Found machine to be added: Dewberry Farm
2024-10-22 00:41:41 INFO Location Texas (26/125): Changes in 3/310 machines found.
2024-10-22 01:00:00 INFO To-be-added-machine Lexington Style Trimmings BBQ     in North Carolina seems unavailable: http://209.221.138.252/Details.aspx?location=59354 with Internal Server Error (500)
2024-10-22 01:02:53 INFO Seems that machine Buc-ee's already exists as: Buc-ee’s
2024-10-22 01:04:03 INFO To-be-added-machine The Big Merino  in Australia seems unavailable: http://209.221.138.252/Details.aspx?location=993 with Internal Server Error (500)
2024-10-22 01:30:02 INFO To-be-added-machine Dierenpark (ZOO) in Netherlands seems unavailable: http://209.221.138.252/Details.aspx?location=708 with Internal Server Error (500)
2024-10-22 01:34:11 INFO 71/77: Found machine to be added: Sete Cidades, Green Love Restaurant, 
2024-10-22 01:34:15 INFO Location Portugal (101/125): Changes in 1/77 machines found.
2024-10-22 01:41:54 INFO 
 Result: 5 changes, 4 new machines found and 1 machines retired
2024-10-22 01:51:22 ERROR Our machine Onze Lieve Vrouw Kathedral  in Belgium from Server shown as available but http://209.221.138.252/Details.aspx?location=401561 responds Internal Server Error (500)
2024-10-22 01:57:36 ERROR Found 6 problems that require manual intervention
2024-10-22 01:57:36 INFO ======Location differ completed at 2024-10-22 01:57:36=======
2024-10-22 01:57:37 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
New problems require attention.

File: data/server_locations.json [hunk 1]
```diff
@@ -66952,11 +66952,11 @@
                 "status": "unvisited",
                 "external_url": "null",
                 "internal_url": "null",
-                "latitude": 42.523759313620914,
-                "longitude": -87.89325782771036,
+                "latitude": "42.523759313620886",
+                "longitude": "-87.89325782771036",
```
Comment (NinaWie) [inline data/server_locations.json:66956]: @jannisborn was it always the case that lat and lon are sometimes floats and sometimes strings?? Seems to be a mix here. Maybe due to merge of all_locations and server_locations?

**Comment by jannisborn** [comment]
2024-10-22 22:30:04 INFO ======Location differ joblog from 2024-10-22 22:30:04=======
2024-10-22 22:32:57 INFO 82/139: Found machine to be added: James A. Garfield House National Historic Site in Ohio
2024-10-22 23:20:28 INFO Mt Washington Cruises Ticket Office (in New Hampshire) is currently unavailable
2024-10-22 23:29:34 INFO Harry Potter Exhibition (in New York) is currently unavailable
2024-10-23 00:14:05 INFO 
 Result: 3 changes, 1 new machines found and 2 machines retired
2024-10-23 00:29:40 ERROR Found 6 problems that require manual intervention
2024-10-23 00:29:40 INFO ======Location differ completed at 2024-10-23 00:29:40=======
2024-10-23 00:29:41 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-10-23 22:30:01 INFO ======Location differ joblog from 2024-10-23 22:30:01=======
2024-10-23 23:30:59 INFO 134/167: Found machine to be added: Norh Carolina State Fairgrounds in North Carolina
2024-10-24 00:12:20 INFO 
 Result: 1 changes, 1 new machines found and 0 machines retired
2024-10-24 00:17:02 ERROR Our machine Scenic Caves Nature Adventures in Canada from Device shown as available but http://209.221.138.252/Details.aspx?location=44724 responds Internal Server Error (500)
2024-10-24 00:27:55 ERROR Found 6 problems that require manual intervention
2024-10-24 00:27:55 INFO ======Location differ completed at 2024-10-24 00:27:55=======
2024-10-24 00:27:56 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-10-24 22:30:02 INFO ======Location differ joblog from 2024-10-24 22:30:02=======
2024-10-24 22:37:08 INFO 146/225: Found machine to be added: Hwy 66 Chevron in Arizona
2024-10-24 23:28:46 WARNING Exception encountered when testing link 'http://209.221.138.252/Details.aspx?location=3544': ("Connection broken: ConnectionResetError(104, 'Connection reset by peer')", ConnectionResetError(104, 'Connection reset by peer'))
2024-10-25 00:12:53 INFO 
 Result: 1 changes, 1 new machines found and 0 machines retired
2024-10-25 00:28:28 ERROR Found 6 problems that require manual intervention
2024-10-25 00:28:28 INFO ======Location differ completed at 2024-10-25 00:28:28=======
2024-10-25 00:28:29 INFO Detected change in server_locations.json - push to github

**Review by NinaWie — APPROVED** [review]

**Pull request merged**